core/src/main/scala/org/apache/spark/internal/config/ConfigBuilder.scala

```diff
@@ -90,6 +90,14 @@ private[spark] class TypedConfigBuilder[T](
     new TypedConfigBuilder(parent, s => fn(converter(s)), stringConverter)
   }
 
+  /** Check that user-provided value for the config match a validator */
```

Comment:
`Check that user-provided value for the config match a validator`
->
`Checks if the user-provided value for the config matches the validator`